plugins/postcss-rebase-url/src/normalized-dir.ts

```diff
@@ -0,0 +1,5 @@
+import path from 'path';
+
+export function normalizedDir(x: string): string {
```

Comment:
Should we add some basic docs here (and serialize-string) as with `rebase`?

Reply:
done